modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/Tabs/BasicInfo/MaxLengthProperties.tsx

```diff
@@ -131,7 +131,7 @@ export function MaxLengthProperties({
 							onSettingsChange({
 								name: 'maxLength',
 								value:
-									Number(value) <= 280
+									Number(value) <= defaultMaxLength
```

Comment:
Good job with this PR @Artartur! The changes made achieved the expected result:


https://github.com/liferay-objects/liferay-portal/assets/61854510/11b68f3a-696a-4a90-9389-835bed551cd2